src/scandeval/cli.py

```diff
@@ -147,6 +147,25 @@
     help="""Whether to skip evaluation of models which have already been evaluated,
     with scores lying in the 'scandeval_benchmark_results.jsonl' file.""",
 )
+@click.option(
+    "--fix-embedding/--no-fix-embedding",
+    "-f/-nf",
+    default=True,
+    show_default=True,
+    help="""Whether to increase the size of the model embedding to the size of the
+            tokenizer vocabulary, if needed."""
+)
+@click.option(
+    "--override-model-config",
+    "-o",
+    default=None,
+    show_default=True,
+    help="""Override model config from HF Hub with revision, framework, task and
+            languages. This parameter is expected to be a dictionary mapping
+            a string defining the model id to a dictionary mapping revision, framework,
+            and task to appropriate strings and languages to a list of strings
+            with language codes."""
+)
```

Comment:
Maybe this should be renamed to `--model-config`. Also, I think it would be a bit more intuitive to use `type=(str, str)` and `multiple=True` as [described here](https://stackoverflow.com/a/72108720/3154226). This would allow the following:

```
scandeval --model-config framework pytorch
```

Reply:
Also, if this is purely for being able to use `scandeval` with local models, I don't think it's necessary to specify `task`, `language` or even `revision`? So maybe this option is not needed at all?

Reply:
True. Language can be set separately. I'll update the documentation for --language when I update the pull request. Same goes for task. Revision could be "local" - what do you think?

Reply:
Ok, after reviewing the code, I found out that the languages and task attributes of modelconfig are not used at all. I just set them to [] and "unknown" for local models.